File: Python/bmimetric.py
def bmi_metric(weight, height):
    return weight/(height**2)


if __name__ == "__main__":
    w = float(input("Please enter weight in kilograms: "))
    h = float(input("Please enter height in meters: "))
    bmi = bmi_metric(w, h)
    print(f"BMI is: {bmi}")

File: C++/bmimetric.cpp
#include <iostream>


int main() {
    double height, weight, bmi;

    std::cout << "Please enter weight in kilograms: ";
    std::cin >> weight;
    std::cout << "Please enter height in meters: ";
    std::cin >> height;

    bmi = weight/(height*height);
    std::cout << "BMI is: " << bmi;
    return 0;
}
